packages/core/lib/command-utils.js

```diff
@@ -22,7 +22,9 @@ const getCommand = ({ inputStrings, options, noAliases }) => {
     return null;
   }
 
-  const firstInputString = inputStrings[0];
+  const firstInputString = ["-v", "--version"].includes(inputStrings[0])
```

Comment:
I feel like this is a slightly confusing way to do this. By that I mean it takes a minute to figure out what the code is attempting to do. You probably Maybe something like the following would be clearer?
```
const firstInputString = inputStrings[0];
if (firstInputString === "-v" || firstInputString === "--version") {
  return {
    // and then explicitly return the whole object here
```

What do you think of that? I will also say that my intuition tells me to avoid messing with `firstInputString` like you are doing just to handle one or two cases that will occur. I'm having trouble explaining exactly why but I don't like swapping out what the user entered and labeling it as `firstInputString`, as in the case where they entered `-v` it isn't the first input string. Let the first input string be the first input string.

Or probably a better idea might be to just add a special case in the `if/else` that is below. Then you can set `chosenCommand` to "version". That is probably a more eloquent way to handle this than what I wrote above.

Reply:
I do agree it is probably a better idea to not mess with `firstInputString`. I will work on moving it to the `if/else` below.